### PR TITLE
Fix wrongful LR scheduler step order warnings

### PR DESCRIPTION
**Pull request by ottonemo**
With PyTorch 1.1.0 the optimizer step order was fixed (see af04b559445a76da33a2750ff630dee2749402d3) and a warning is issued when the order is violated. Since some LR scheduler tests don't call the optimizer at all this warning is falsely issued. We can fix this by calling `optimizer.step` anyway.

Relates to #467 and #561.

### CODE REVIEW

**Comment by BenjaminBossan** [comment]
Thank you very much.
